### PR TITLE
Accept command line parameter to override predefined variables

### PR DESCRIPTION
**Pull request by kenhys**

### CODE REVIEW

File: powershell/create-mrnzip.ps1 [hunk 1]
```diff
@@ -1,21 +1,38 @@
-. ".\versions.ps1"
+Param(
+  [Parameter(mandatory=$false)][String]$mariadbVersion = $null,
+  [Parameter(mandatory=$false)][String]$mroongaVersion = $null,
+  [Parameter(mandatory=$false)][String]$platform = $null
```
Comment (cosmo0920) [inline powershell/create-mrnzip.ps1]: Can we use `[String[]]` type?

File: powershell/install-mroonga.ps1 [hunk 2]
```diff
@@ -1,8 +1,27 @@
 # @$mariadbVer MariaDBVer
 # @$arch win32/winx64
 # @$installSqlDir specify install.sql dir
+
+Param(
+  [Parameter(mandatory=$false)][String]$mariadbVersion = $null,
+  [Parameter(mandatory=$false)][String]$mroongaVersion = $null,
+  [Parameter(mandatory=$false)][String]$platform = $null
```
Comment (cosmo0920) [inline powershell/install-mroonga.ps1]: ditto.

**Review by cosmo0920 — CHANGES_REQUESTED** [review]
I've reviewed your PR. 
Could you take a look review comments?

File: powershell/create-mrnzip.ps1 (hunk 1 above)
Comment (kenhys) [reply powershell/create-mrnzip.ps1]: Fixed.

File: powershell/install-mroonga.ps1 (hunk 2 above)
Comment (kenhys) [reply powershell/install-mroonga.ps1]: Fixed.

**Comment by kenhys** [comment]
@cosmo0920  I've fixed to use string[], could you merge this PR?

**Comment by cosmo0920** [comment]
I'm confirming on my local environment.
Please wait a bit. :cat:

File: powershell/create-mrnzip.ps1 [hunk 3]
```diff
@@ -1,21 +1,36 @@
...
   $compressionLevel = [System.IO.Compression.CompressionLevel]::Optimal
   $includeBaseDir = $false
   [System.IO.Compression.ZipFile]::CreateFromDirectory($srcDir, $destFile, $compressionLevel, $includeBaseDir)
 }
 
-$platform = "win32", "winx64"
 cd $workDir
 
-foreach ($arch in $platform)
+foreach ($arch in $platforms)
```
Comment (cosmo0920) [inline powershell/create-mrnzip.ps1:30]: :+1:

File: powershell/install-mroonga.ps1 [hunk 4]
```diff
@@ -34,10 +51,9 @@ function Install-Mroonga($mariadbVer, $arch, $installSqlDir) {
 
 $installSqlDir = ".\share\mroonga"
 
-$platform = "win32", "winx64"
-foreach ($arch in $platform)
+foreach ($arch in $platforms)
```
Comment (cosmo0920) [inline powershell/install-mroonga.ps1:54]: :+1:

**Review by cosmo0920 — APPROVED** [review]
Looks good!
Thank you. :smile_cat:

**Pull request merged**